groups/nts/ntscfg/ntscfg_platform.h

```diff
@@ -217,9 +217,9 @@ struct Platform {
     static int exit();
 
     /// Return true if the version of the operating system running the current
-    /// process supports timestamping incoming and outgoing data, otherwise
-    /// return false.
-    static bool supportsTimestamps();
+    /// process supports asynchronous socket notifications via some mechanism
+    /// such as the Linux error queue.
+    static bool supportsNotifications();
```

Comment:
That sounds sort of strange when it is at nts level. Concept of _notifications_ was introduced on Reactor level, which is NTC.

Reply:
I have removed `ntscfg::Platform::supportsTimestamps` in favor of a new function, `ntsu::SocketOptionUtil::supportsTimestamps(socket)`. We need low-level support for platform capabilities, because even if the high-level interface is in NTC the low-level mechanics are requested from NTS.